docs/sources/send-data/otel/native_otlp_vs_loki_exporter.md

```diff
@@ -0,0 +1,93 @@
+---
+title: How is native OTLP endpoint different from Loki Exporter
+menuTitle:  Native OTLP endpoint vs Loki Exporter
+description: Native OTLP endpoint vs Loki Exporter
+weight:  251
+---
+
+# Introduction
+
+OTel is quickly becoming an industry standard with increasing adoption. Prior to the Loki 3.0 release, there was no native support for ingesting OTel logs to Loki, which led to creation of [LokiExporter](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/exporter/lokiexporter/README.md). While the LokiExporter got the job done of ingesting OTel logs to Loki, it did not provide a native user experience, and the querying experience was not as optimal. As part of our effort to improve user experience with OTel, we added native OTel log ingestion support to Loki with 3.0 release.
+
+# What has changed?
+
+## Formatting of logs
+
+[LokiExporter's README](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/exporter/lokiexporter/README.md) explains in detail the formatting of OTel logs ingested via LokiExporter, while Loki’s native OTel log ingestion endpoint is described in detail in its documentation. Here is a summary of how the logs are formatted between the two:
+
+### LokiExporter
+
+- Index Labels:
+  - It supports label control via hints set by the OTel client/collector.
+  - Default labels:
+    - job=service.namespace/service.name 
+    - instance=service.instance.id 
+    - exporter=OTLP 
+    - level=severity
+- Log Body: Encodes log records and attributes in json(default) or logfmt format.
+
+### Loki’s native OTel log ingestion endpoint:
```

Comment:
I'm not sure that this needs to be a heading, since it's really part of the "Formatting of Logs" section.

Reply:
Do we also want to list default labels here? If you're comparing the two, I'd list the same things in each list.

Reply:
> I'm not sure that this needs to be a heading, since it's really part of the "Formatting of Logs" section.

I have updated the document so that it does not have it as a heading and made it look bold to stand out.

> Do we also want to list default labels here? If you're comparing the two, I'd list the same things in each list.

To avoid having to maintain the list at two places, I have added a link to the parent doc where we have the list.

Reply:
That's an excellent solution.